CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+-   Please provide as much context as possible and describe the _reproduction steps_ that someone else can follow to recreate the issue on their own. This usually includes your code. For good bug reports you should isolate the problem and create a reduced test case.
+-   Provide the information you collected in the previous section.
+
+Once it's filed:
+
+-   The project team will label the issue accordingly.
+-   A team member will try to reproduce the issue with your provided steps. If there are no reproduction steps or no obvious way to reproduce the issue, the team will ask you for those steps and mark the issue as `needs-repro`. Bugs with the `needs-repro` tag will not be addressed until they are reproduced.
+-   If the team is able to reproduce the issue, it will be marked `needs-fix`, as well as possibly other tags (such as `critical`), and the issue will be left to be [implemented by someone](#your-first-code-contribution).
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion for Backtrace Javascript SDK, **including completely new features and minor improvements to existing functionality**. Following these guidelines will help maintainers and the community to understand your suggestion and find related suggestions.
+
+<!-- omit in toc -->
+
+#### Before Submitting an Enhancement
+
+-   Make sure that you are using the latest version.
+-   Read the relevant SDK/tooling documentation carefully and find out if the functionality is already covered, maybe by an individual configuration.
+-   Perform a [search](/issues) to see if the enhancement has already been suggested. If it has, add a comment to the existing issue instead of opening a new one.
+-   Find out whether your idea fits with the scope and aims of the project. It's up to you to make a strong case to convince the project's developers of the merits of this feature. Keep in mind that we want features that will be useful to the majority of our users and not just a small subset. If you're just targeting a minority of users, consider writing an add-on/plugin library.
+
+<!-- omit in toc -->
+
+#### How Do I Submit a Good Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](/issues).
+
+-   Use a **clear and descriptive title** with valid prefix for the issue to identify the suggestion.
+    -   We use **squash commits** when merging pull requests, so the title should resemble a [commit message](#commit-messages)
+-   Provide a **step-by-step description of the suggested enhancement** in as many details as possible.
+-   **Describe the current behavior** and **explain which behavior you expected to see instead** and why. At this point you can also tell which alternatives do not work for you.
+-   **Explain why this enhancement would be useful** to most Backtrace Javascript SDK users. You may also want to point out the other projects that solved it better and which could serve as inspiration.
```

Comment:
Can we drop it and focus on the real reason why we want to have CONTRIBUTING document?